yt/data_objects/data_containers.py

```diff
@@ -1006,7 +1006,7 @@ def max(self, field, axis=None):
                 return rv[0]
             return rv
         elif axis in self.ds.coordinates.axis_name:
-            r = self.ds.proj(field, axis, data_source=self, method="mip")
+            r = self.ds.proj(field, axis, data_source=self, method="max")
             return r
```

Comment:
While you're at it, let's simplify this :)
```suggestion
            return self.ds.proj(field, axis, data_source=self, method="max")
```